lib/build/definitions/application.js

```diff
@@ -100,6 +100,11 @@ export default function({project, taskUtil, getTask}) {
 				}, Promise.resolve());
 			}
 		});
+	} else {
+		// No bundles defined. Just set task so that it can be referenced by custom tasks
+		tasks.set("generateBundle", {
```

Comment:
There's also a corresponding task in `lib/build/definitions/library.js`. 
Should we also do the same there?